yubihsm-container/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# SPDX-FileCopyrightText: 2023 Technology Innovation Institute (TII)
+# SPDX-License-Identifier: Apache-2.0
+
+FROM ubuntu:jammy
```

Comment:
FYI: Latest LTS is ubuntu:noble, and docker images for it exist already. Not sure if you want to use it already.

Reply:
There seems to be issues with package contents. Had to change quite a bit. Perhaps we can leave jammy and research which image to use as a side-track (combining this comment with https://github.com/tiiuae/ci-yubi/pull/1#discussion_r1599669487 )? Would that work?

Reply:
Yes, jammy is still supported for three more years, so we have time. This was just a suggestion in case you *want* to go to noble already.